types/index.d.ts

```diff
@@ -373,11 +382,7 @@ export interface PagesOptions {
  * [`Provider`](https://next-auth.js.org/getting-started/client#provider) |
  * [`session` callback](https://next-auth.js.org/configuration/callbacks#jwt-callback)
  */
-export interface Session extends Record<string, unknown> {
-  user?: User
-  accessToken?: string
-  expires: string
-}
+export interface Session extends Record<string, unknown>, DefaultSession {}
```

Comment:
I think that the `Session` and `JWT` interfaces shouldn’t extend `Default{Session,JWT}` as that nullifies the point of this change, unfortunately.

Reply:
![image](https://user-images.githubusercontent.com/18369201/115619643-6da23900-a2f4-11eb-95cf-3cbdf2cfb562.png)
So here is how it looks like for me, when I override `Session` (in the middle)

Reply:
That worked like charm, thank you! 🙌